.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ defaults: &defaults
     # Note: when updating the docker image version,
     #       make sure there are no extra old versions lying around.
     #       (e.g. `rg -F --hidden <old_tag>`)
-    - image: pyodide/pyodide-env:20250311-chrome134-firefox136-py313
+    - image: zsaladin/pyodide-selenium-manager:dev2
```

Comment:
It's great that the size of the image is ~100 MB smaller than the previous one.

<img width="997" height="193" alt="image" src="https://github.com/user-attachments/assets/356641ec-1594-482c-bfa1-f4be50ad09c8" />

<img width="759" height="212" alt="image" src="https://github.com/user-attachments/assets/4ed482f8-5279-4480-855b-27e3fcbe8599" />